### ISSUE TITLE
Accessibility of WebAuthN Dialogs

### ISSUE THREAD

**Issue by yao-ding** [w3c/webauthn#1672]
Hi WebAuthn WG,

I'm Yao Ding, an Accessibility Research at Facebook.  I’m representing the Consumer Deployment WG of [FIDO Alliance](https://fidoalliance.org/) to seek clarification on WebAuthn dialog accessibility.

How does the WebAuthn WG ensure that WebAuthn dialogs are accessible?  We assume the dialogs are rendered as browser pop-ups so their accessibility depends on that of the browser's default pop-up, and the popular browsers like Chrome Safari and Edge should be fairly accessible.   Is that correct?

Do you have any accessibility requirements for browser vendors?  Or have you done any accessibility evaluations on WebAuthn dialogs?

Thanks!
Yao

**Comment by equalsJeffH** [comment]
on 2021-10-06 call: 
@ve7jtb gave background wrt this issue: Some FIDO CDWG members have raised questions such as "we cannot turn off the alternative login means the user has in favor of webauthn because they do not know whether the webauthn authenticator and client platform impls support appropriately accessible modalities for the situation at hand". discussion ended up with @agl signed up to craft an answer to this issue.

**Comment by agl** [comment]
Browser UI is generally out of scope for W3C specifications. Speaking for Chrome, our WebAuthn UI undergoes internal accessibility review and we hope that it is broadly accessible. If there are accessibility concerns then one should raise an issue in the usual way, at https://crbug.com.

Outside of the browser we wish for there to be a vibrant market of authenticators so that each user is able to purchase an authenticator that meets their needs.

**Comment by yao-ding** [comment]
@equalsJeffH @agl thank you so much for your answers!  

@sbweeden  closing the issue as we didn't have further questions from last FIDO meeting.
